src/static_analysis.rs

```diff
@@ -119,6 +119,7 @@ impl Default for CfgNode {
 }
 
 /// Result of the executable analysis
+#[derive(Debug)]
```

Comment:
Necessary?

Reply:
Thanks, it's not necessary now, I will remove it. :)

Reply:
Done.